internal/resources/aws/cloudwatch_metric_alarm.go

```diff
@@ -0,0 +1,87 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+
+	"fmt"
+
+	"github.com/shopspring/decimal"
+)
+
+type CloudwatchMetricAlarm struct {
+	Address            string
+	Region             string
+	ComparisonOperator string
+	MetricCount        int64
```

Comment:
```suggestion
	Metrics        int64
```
Not  got a strong opinion on this, this is just how we handle this case elsewhere.

Reply:
Updated 👍